SGoopas/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.3.0f3
```

Comment:
don’t commit this change please

Reply:
Twice?

Reply:
What? This comment is about ProjectVersion.txt. The twice thing is about the .Unity files